engine/apps/api/views/channel_filter.py

```diff
@@ -109,24 +112,10 @@ def perform_update(self, serializer):
     @action(detail=True, methods=["put"])
     def move_to_position(self, request, pk):
         instance = self.get_object()
-        position = get_move_to_position_param(request)
-
         if instance.is_default:
             raise BadRequest(detail="Unable to change position for default filter")
 
-        prev_state = instance.insight_logs_serialized
```

Comment:
Changing `order` is now handled by the parent class [OrderedModelViewSet](https://github.com/grafana/oncall/pull/2568/files#diff-eb62521ccbcb072d1bd2156adeadae3b5895bce6d0d54432a23db3948b0ada54R11-R34)